index.js

```diff
@@ -12,9 +12,11 @@ function instance(system, id, config) {
 	self.request_id = 0;
 	self.stash = [];
 	self.command = null;
-
+	
 	self.video_sources = [
-		{ id: 'SDI', label: 'SDI' },
```

Comment:
this line is not needed. the next one covers the SDI input for Teranex Express.

Reply:
That is not an accurate statement (at least according to the manual) ... Teranex 2D and 3D use "SDI" as a source because they only have the one where-as the AV uses "SDI1" and "SDI2" because it has 2.

Reply:
in this case ( i don't have 3D) I suggest to erase the line with the Express remark and stay with an SDI option for 2D, 3D and Express. Teranex AV users can use SDI1 and SDI2 options.

What do you say @krocheck ?

Reply:
So I guess what we really ought to do is have the user select their model in the instance config and then modify the available sources depending on the model selected.

Reply:
if there is an option for this then yes.

Reply:
Ugh ... I can probably do it.  My code from other BMD devices should let us auto-detect.

Reply:
@krocheck if its too much work than I think the current method is more than enough for now. maybe for a later update you can put in an auto-detect feature.

Reply:
Yeah, I have no time for this.  We'll proceed as-is without model selection.  Maybe I'll circle back and do that later.